src/main/java/frc/robot/Robot.java

```diff
@@ -19,9 +23,22 @@
  * project.
  */
 public class Robot extends TimedRobot {
-  
+     
+     @Override
      public void teleopPeriodic(){
 
+          //PixyCameraConnector objectPixyCameraConnector = new PixyCameraConnector();
+          VisionFieldLayoutRecognizer objectVisionFieldLayoutRecognizer = new VisionFieldLayoutRecognizer();
+          PixyCameraConnector objPixyCameraConnector = new PixyCameraConnector();
+
+          SmartDashboard.putNumber("Number Found", 11);
+
+          ArrayList<FieldVisionInput> arraylistFieldVisionInput = 
+                    objPixyCameraConnector.getFieldVisionInput();
+
+          objectVisionFieldLayoutRecognizer.detectPose(arraylistFieldVisionInput);
```

Comment:
dont we need to do something with the result of all our hard work?

Reply:
understood